config.yaml

```diff
@@ -1,8 +1,42 @@
 baseURL: "https://jellyfin.org"
-languageCode: en-us
+# languageCode: en-us
+DefaultContentLanguage: en-us
+enableMissingTranslationPlaceholders: true
+languages:
+  pt-Br:
+    title: "Jellyfin: O sistema de mídia de software livre"
+    languageName: Português (Brasil)
+    weight: 2
+    contentDir: "content/pt-br"
+    params:
```

Comment:
Can we add `$ref` elements here or something? Separate files for each language would be the best option to get weblate support.